x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -127,6 +128,7 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
 
   private lists: ListPluginSetup | undefined; // TODO: can we create ListPluginStart?
   private licensing$!: Observable<ILicense>;
+  private policyWatcher?: PolicyWatcher;
```

Comment:
Why optional?

Reply:
not populated at construction -- so typescript yelled at me.

Reply:
Ahhh.. Yes, now I remember :) thanks